ownership-chain/e2e-tests/tests/test-create-collection.ts

```diff
@@ -28,25 +36,20 @@ describeWithExistingNode("Frontier RPC (Create Collection)", (context) => {
         this.timeout(70000);
 
         const collectionId = "0";
-        let nonce = await context.web3.eth.getTransactionCount(GENESIS_ACCOUNT);
-        
-        await context.web3.eth.accounts.wallet.add(GENESIS_ACCOUNT_PRIVATE_KEY);
-        const result = await contract.methods.createCollection(GENESIS_ACCOUNT).send({ from: GENESIS_ACCOUNT, gas: "0x10000", nonce: nonce++ });
+
+        const result = await contract.methods.createCollection(GENESIS_ACCOUNT).send({ from: GENESIS_ACCOUNT, gas: GAS, nonce: nonce++ });
```

Comment:
what is the advantage of creting a one-line utility function that has the same signature as the original function?

Reply:
this part looks a bit ugly when it's repeated. but it might be just my preference, so feel free to ignore it

```
send({ from: GENESIS_ACCOUNT, gas: GAS, nonce: nonce++ });
```